insights/combiners/httpd_conf.py

```diff
@@ -235,9 +237,64 @@ def get_active_setting(self, directive, section=None):
         if section is not None:
             if values_list:
                 for i, val in enumerate(values_list):
+                    # From each section, preserve only the last ParsedData
+                    # {(section, ""): [ParsedData, ParsedData]} ---> ParsedData
                     values_list[i] = val.values()[0][-1]
                 return values_list
             return []
         else:
             if values_list:
                 return values_list[-1]
+
+    def get_section_list(self, section):
```

Comment:
Just a note that it is a partially inspired by `get_setting_list` because it is quite complex matter and this approach helped me to not get lost.